node/src/command.rs

```diff
@@ -84,17 +91,98 @@ pub fn run() -> Result<()> {
 
                 runner.sync_run(|config| cmd.run::<Block, Executor>(config))
             } else {
-                println!(
-                    "Benchmarking wasn't enabled when building the node. \
+                Err("Benchmarking wasn't enabled when building the node. \
```

Comment:
I wanted to test running the benchmarking command, but i clearly forgot to build the node with that feature.
I tried to pass that argument at the root like so:

`cargo build --release --features runtime-benchmarks`

But i get error:

```
error: --features is not allowed in the root of a virtual workspace
note: while this was previously accepted, it didn't actually do anything
```

What is the correct way to build the node with the benchmarking feature?

Reply:
Try to run it in the node folder.